### ISSUE TITLE
How to use this repository?

### ISSUE THREAD

**Issue by Mr47121836** [hcy71o/SNAC#5]
I have reviewed the paper, which asserts that there is no audio or information from unknown speakers during the training process. However, in your code, your validation set includes information from unknown speakers. Doesn't this imply that the model had already assimilated information from unknown speakers during training, and the reference speech used for inference corresponds to the speech encountered during training?

![image](https://github.com/hcy71o/SNAC/assets/66686828/0d4cc9ef-6f82-4fc3-939c-cd6c7e31c29b)
![image](https://github.com/hcy71o/SNAC/assets/66686828/6716bc25-01c5-448f-8d8b-27b14c61e706)
For speaker p261，the model already meet this speaker？

please help me！

**Comment by hcy71o** [comment]
First of all, I am not the author of the paper. The purpose of this repo is to test the model, and the authors’ implementation and dataset configuration may differ. Additionally, in the current code, there is no bias in tuning the model because only a single test sample is generated ([link](https://github.com/hcy71o/SNAC/blob/620b1e27bdc2126d9105819242d3640b4834bfdb/train.py#L248-L255)), and there is no validation loss calculation during the training process.

**Comment by Mr47121836** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
